sqlx-cli/src/lib.rs

```diff
@@ -13,6 +13,14 @@ mod prepare;
 pub use crate::opt::Opt;
 
 pub async fn run(opt: Opt) -> anyhow::Result<()> {
+    let path = env::current_dir()?.join("Cargo.toml");
+    if !path.exists() {
```

Comment:
No need to construct an absolute path.

```suggestion
    if !Path::new("Cargo.toml").exists() {
```

Reply:
Thanks for the comment. updated.